packages/components/src/components/primitives/Link/Link.stories.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
```

Comment:
Could we iterate over the properties/variants to avoid manual updates? Check `Button` stories for reference